fixtures/stairville/wild-wash-648-led-rgb-dmx.json

```diff
@@ -0,0 +1,383 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Wild Wash 648 LED RGB DMX",
+  "shortName": "WildWash648LEDRGB",
```

Comment:
```suggestion
```

I'm not convinced this is helpful, when it's basically as long as the full name.